src/main/cpp/domconfigurator.cpp

```diff
@@ -840,13 +846,6 @@ spi::ConfigurationStatus DOMConfigurator::doConfigure
 {
 	m_priv->repository = repository ? repository : LogManager::getLoggerRepository();
 	m_priv->repository->setConfigured(true);
-	if (LogLog::isDebugEnabled())
```

Comment:
do we print this message out(what file is being used for configuration) anyplace else?  That can be useful to know when you're unsure which file is being used

Reply:
Yes, a few lines below it outputs either "Loading configuration file [" ... or "Could not read configuration file [" ...